wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -688,6 +693,12 @@ public static class Config extends PluginConfig {
     @Nullable
     private String preconditionLanguage;
 
+    @Name(NAME_EXECUTION_LANGUAGE)
```

Comment:
Rather that `EXECUTION_LANGUAGE`, I believe `USE_SQL_ENGINE` would make more sense.